build/symlink-dependencies/index.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
```

Comment:
This addon seems unused?  Did you intend to add `ember-addon: { paths: ['./build/symlink-dependencies']}` to the `package.json`?